test-app/build-tools/android-metadata-generator/src/src/com/telerik/metadata/ClassDirectory.java

```diff
@@ -82,16 +78,13 @@ private static NativeClassDescriptor getClassDescriptor(String name, File file)
                     if ("Lkotlin/Metadata;".equals(annotationType)) {
                         MetadataAnnotation kotlinClassMetadataAnnotation = new BytecodeMetadataAnnotation(annotationEntry);
                         NativeClassDescriptor kotlinClassDescriptor = new KotlinClassDescriptor(javaClass, kotlinClassMetadataAnnotation);
-                        isKotlinClass = true;
                         analyticsCollector.markHasKotlinRuntimeClassesIfNotMarkedAlready();
                         return kotlinClassDescriptor;
                     }
                 }
             }
 
-            if (!isKotlinClass) {
-                return new JavaClassDescriptor(javaClass);
-            }
+            return new JavaClassDescriptor(javaClass);
```

Comment:
I believe this is only supposed to occur on a Kotlin class; which is why it is wrapped in the if statement.

Reply:
The IDE complained thats its always false, which im agreeing on.

Reply:
You are correct, I totally missed the "return on line 87/82"   Duh.  :D